samples/LargeResponseApp/LargeResponseApp.csproj

```diff
@@ -11,4 +11,8 @@
     <ProjectReference Include="..\..\src\Kestrel\Kestrel.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(MicrosoftNETCoreCompilersVersion)" PrivateAssets="All" />
```

Comment:
Is this just for consistency within the repro? Or does these mean all consumers of Kestrel would need a new compiler?